ckanext/dcatapchharvest/tests/test_swiss_schemaorg_profile_serialize.py

```diff
@@ -184,3 +190,36 @@ def test_graph_from_dataset(self):
             eq_(len([t for t in g.triples((dataset_ref, item[1], None))]), len(values))
             for value in values:
                 assert self._triple(g, dataset_ref, item[1], item[2](value))
+
+    # new test
```

Comment:
This comment is not very informative. Please remove it.

Reply:
done